src/test/java/com/algorand/algosdk/integration/Stepdefs.java

```diff
@@ -158,6 +158,59 @@ public SignedTransaction signWithAddress(Transaction tx, Address addr) throws co
         return acct.signTransaction(tx);
     }
 
+    /**
+     * advanceRound is a convenience method intended for testing with DevMode networks.
+     * <p>
+     * Since DevMode block generation requires a transaction rather than time passing, test assertions may require advancing rounds.  advanceRound issues advanceCount payments to advance rounds.
+     */
+    private void advanceRoundsV1(int advanceCount) {
+        initializeDevModeAccount();
+        for (int i = 0; i < advanceCount; i++) {
+            try {
+                Transaction tx =
+                        Transaction.PaymentTransactionBuilder()
+                                .sender(devMode.getAddress())
+                                .suggestedParams(acl.transactionParams())
+                                .amount(BigInteger.valueOf(1))
+                                .receiver(devMode.getAddress())
+                                .build();
+                SignedTransaction st = devMode.signTransaction(tx);
+                acl.rawTransaction(Encoder.encodeToMsgPack(st));
+            } catch (Exception e) {
+                throw new RuntimeException(e);
+            }
+        }
+    }
+
+    private Account devMode;
```

Comment:
Could we move this up the above function if we intend to use it as global/static state?

Reply:
@algochoi Sure - How does https://github.com/algorand/java-algorand-sdk/pull/350/commits/d0aa73332b7e81035892c81494edffa48e12c9af look?